### ISSUE TITLE
Building on top of the trained model

### ISSUE THREAD

**Issue by MohameHassan** [ildoonet/tf-pose-estimation#93]
Hi,
I would like to use the trained CMU network as part of my network architecture. I don't want to retrain a pose estimator, but i want to use the trained network and add a few convolution and FC layers on top of it to generate something else. I would like  to train for the weights of the new layers only. 
Now I am facing the following problems:
- The first part of my network just load the pretrained graph. I did the same as `class TfPoseEstimator' which i think loads the optimized graph. The problem is that the input to this graph is of size (1,w,h,3) so I can't use it to train batches. How can I avoid that? 
- I can't see in the training code where is the data being fed. It is clear for the case of the validation set but not for the training set. How can i feed the data from the queue to my network descried above.

**Comment by ildoonet** [comment]
@MohameHassan 

- The frozen graphs are made with batch size = one. You can use weights with more batch, but you need to create a network and copy weights. 

- Training data are fed by another thread. See: https://github.com/ildoonet/tf-pose-estimation/blob/master/src/train.py#L74

**Comment by MohameHassan** [comment]
Hi @ildoonet  
I tried to follow the training thread but it's still not clear to me how exactly is the data being fed.
I built a different network with input layer lets call it `X` now how can i make use of the training thread to feed the data to this input layer `X` ?
I am looking for a line of code like this https://github.com/ildoonet/tf-pose-estimation/blob/8543abfc00a7ff149f64ce84114a3148a93b6074/src/train.py#L238
But  I can't find it

**Comment by ArtificialNotImbecile** [comment]
@ildoonet Although this issue is closed, I think "The frozen graphs are made with batch size = one" is not correct since I visualize the frozen graph via tensorboard, and the image placeholder is `?x?x?x3`. 
![tb-vs](https://user-images.githubusercontent.com/10048576/54729229-42047000-4bbd-11e9-89b6-4c9bfc964939.png)
